hoomd/hpmc/update.py

```diff
@@ -949,9 +951,9 @@ class QuickCompress(_Updater):
         L_current``
 
     Tip:
-        Use the MoveSizeTuner (TODO: make reference) in conjunction with
-        `QuickCompress` to adjust the move sizes to maintain a constant
-        acceptance ratio as the density of the system increases.
+        Use the `MoveSizeTuner` in conjunction with `QuickCompress` to adjust
```

Comment:
Have we decided to use shorter links like `MoveSizeTuner` or longer ones like `hoomd.hpmc.tune.MoveSizeTuner`. I am fine either way, but I have been using the longer format.

Reply:
Explicit is better than implicit and the full namespace is copy and pastable into user scripts. We should use the full namespace.

Reply:
I do use the class name only when referring to the class currently being documented. I think that is fine given the local context.